modules/ngap_module/NgapApi.cc

```diff
@@ -79,6 +79,17 @@ namespace ngap {
             "kNumberType"
     };
 
+
+    NgapApi::NgapApi() : d_cmr_endpoint_url(DEFAULT_CMR_ENDPOINT_URL) {
+        bool found;
+        string key_value;
+        TheBESKeys::TheKeys()->get_value(NGAP_CMR_ENDPOINT_URL, key_value, found);
```

Comment:
No need to change this code - it works - but there are other functions in TheBESKeys that are easier to use.

Reply:
I bam not aware of these. All of the examples of this that I I have found are of this form.